great_expectations/checkpoint/util.py

```diff
@@ -207,26 +207,11 @@ def send_email(
 # TODO: <Alex>A common utility function should be factored out from DataContext.get_batch_list() for any purpose.</Alex>
```

Comment:
Is this TODO still valid?

Reply:
@donaldheppner Unfortunately, this comment is still valid -- we have not had a chance to do the work.  I would like to leave it for now and remove only after we had a chance to go through the backlog formally.  Thank you.